CommonMark/Syntax/InlineTag.cs

```diff
@@ -1,13 +1,11 @@
 ﻿using System;
-using System.Collections.Generic;
-using System.Text;
 
 namespace CommonMark.Syntax
 {
     /// <summary>
     /// Specifies the element type of an <see cref="Inline"/> instance.
     /// </summary>
-    public enum InlineTag
+    public enum InlineTag : byte
```

Comment:
That was before `Header`, which I believe (provided that JIT optimizes `Block` properly) now shares a word with `BlockTag`.